clusterloader2/testing/density/constraints_100.yaml

```diff
@@ -0,0 +1,24 @@
+coredns:
+  cpuConstraint: 0.01
+  memoryConstraint: 9437184 #9 * (1024 * 1024)
+fluentd-gcp:
+  cpuConstraint: 1
+  memoryConstraint: 220200960 #210 * (1024 * 1024)
+heapster:
+  cpuConstraint: 0.1
+  nemoryConstraint: 340787200 #325 * (1024 * 1024)
+kube-apiserver:
+  cpuConstraint: 1.7
+  memoryConstraint: 975175680 #930 * (1024 * 1024)
```

Comment:
This is really small comparing to this one: https://github.com/kubernetes/kubernetes/blob/master/test/e2e/scalability/density.go#L177
Are we sure it's fine?

Reply:
Ohh - I think I may understand it now - in ClusterLoader we are *always* running density before load. So that may explain some differences, I think.

If so, it's probably worth adding a comment somewhere (that the constraints assume that nothing was really running before this test).

Reply:
Do we want to find constraints for every density test run or just for our ci job density test?

TBH I prefer the solution when we set default constraints to empty and set constraints in ci job definition, so this will be job specific.